src/vs/workbench/contrib/notebook/browser/contrib/outline/notebookOutline.ts

```diff
@@ -602,25 +660,26 @@ registerAction2(class ToggleCodeCellEntries extends Action2 {
 	}
 });
 
-registerAction2(class ToggleNonHeaderMarkdownCells extends Action2 {
+registerAction2(class ToggleCodeCellSymbolEntries extends Action2 {
 	constructor() {
 		super({
-			id: 'notebook.outline.toggleNonHeaderMarkdownCells',
-			title: localize('toggleNonHeaderMarkdownCells', "Toggle Non-Header Markdown Cells"),
+			id: 'notebook.outline.toggleCodeCellSymbols',
+			title: localize('toggleCodeCellSymbols', "Code Cell Symbols"),
 			f1: false,
 			toggled: {
-				condition: ContextKeyExpr.equals('config.notebook.outline.showNonHeaderMarkdownCells', true)
+				condition: ContextKeyExpr.equals('config.notebook.outline.showCodeCellSymbols', true)
 			},
 			menu: {
 				id: MenuId.NotebookOutlineFilter,
-				group: 'filter',
+				order: 2,
+				group: '1_code_cells',
 			}
 		});
 	}
 
 	run(accessor: ServicesAccessor, ...args: any[]) {
 		const configurationService = accessor.get(IConfigurationService);
-		const showNonHeaderMarkdownCells = configurationService.getValue<boolean>('notebook.outline.showNonHeaderMarkdownCells');
-		configurationService.updateValue('notebook.outline.showNonHeaderMarkdownCells', !showNonHeaderMarkdownCells);
+		const showCodeCellSymbols = configurationService.getValue<boolean>('notebook.outline.showCodeCellSymbols');
+		configurationService.updateValue('notebook.outline.showCodeCellSymbols', !showCodeCellSymbols);
```

Comment:
Maybe we can create consts for all outline symbols settings other than using string literals.

Reply:
resolved in latest commit. We now have NotebookSetting.xyz for these settings